lib/openid_connect/endpoints/endpoint.rb

```diff
@@ -0,0 +1,41 @@
+module OpenidConnect
+  module Endpoints
+    class Endpoint
+      attr_accessor :app, :user, :client, :redirect_uri, :response_type,
+                    :scopes, :_request_, :request_uri, :request_object
+      delegate :call, to: :app
+
+      def initialize(current_user)
+        @user = current_user
+        @app = Rack::OAuth2::Server::Authorize.new do |req, res|
+          build_attributes(req, res)
+          if OpenidConnect::OAuthApplication.available_response_types.include?(
+              Array(req.response_type).map(&:to_s).join(" "))
+            handle_response_type(req, res)
+          else
+            req.unsupported_response_type!
+          end
+        end
+      end
+
+      def build_attributes(req, res)
+        build_client(req)
+        build_redirect_uri(req, res)
+      end
+
+      def handle_response_type(req, res)
```

Comment:
Unused method argument - `req`. If it's necessary, use `_` or `_req` as an argument name to indicate that it won't be used. You can also write as `handle_response_type(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `res`. If it's necessary, use `_` or `_res` as an argument name to indicate that it won't be used. You can also write as `handle_response_type(*)` if you want the method to accept any arguments but don't care about them.